File: 2025-1/sesion-de-clase-22/SoftProg/SoftProgNegocio/src/main/java/pe/edu/pucp/inf30/softprog/bo/IEmpleadoBO.java
package pe.edu.pucp.inf30.softprog.bo;

import pe.edu.pucp.inf30.softprog.model.rrhh.Empleado;

/**
 *
 * @author eric
 */
public interface IEmpleadoBO extends IBaseBO<Empleado> {
    Empleado buscarPorDni(String dni);
}

File: 2025-2/sesion-de-clase-11/SoftProgNegocio/BO/IEmpleadoBO.cs
using PUCP.SoftProg.Modelo.RRHH;

namespace PUCP.SoftProg.Negocio.BO {
    public interface IEmpleadoBO : IBaseBO<Empleado> {
        Empleado BuscarPorDni(string dni);
    }
}
